app/containers/Applications/ApplicationWizardConfirmation.tsx

```diff
@@ -212,11 +212,15 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
       <br />
       {revision.certificationSignatureIsValid ? (
         <>
-          <h5>
-            Thank you for reviewing the application information. The Certifier
-            has signed off on this application. You may now submit the
-            application.
-          </h5>
+          <div>
+            “By submitting the application the applicant agrees that the
+            information contained on this application, or information contained
+            in emission reports under the Greenhouse Gas Industrial Reporting
+            and Control Act, may be disclosed to British Columbia government
+            employees, contractors and agencies for the purpose of administering
+            the CleanBC Program for Industry or the Greenhouse Gas Industrial
+            Reporting and Control Act”.
```

Comment:
Probably don't need the quotes around the text

Reply:
Updated